src/resources/assets/js/volumes/components/geoMapModal.vue

```diff
@@ -63,21 +102,130 @@ export default {
                 this.disabled = true;
             }
         },
+        toggleActive(id) {
+            if(this.activeIds.includes(id)) {
+                let index = this.activeIds.indexOf(id);
+                this.activeIds.splice(index, 1);
+            } else {
+                this.activeIds.push(id);
+            }
+        },
     },
-    created() {
+    async created() {
         // show the modal upon trigger-event
         this.startLoading();
         this.show = true;
         // get all image + coordinate information from volume-images
-        CoordApi.get({id: this.volumeId})
+        await CoordApi.get({id: this.volumeId})
             .then(response => this.images = response.body, this.handleErrorResponse)
             .finally(this.finishLoading);
-    },
+
+        // provide overlay-url template string
+        await GeoApi.getGeoTiffOverlayUrlTemplate({id: this.volumeId})
+            .then((response) => {
+                this.overlayUrlTemplate = response.body;
+            });
```

Comment:
resolved beda6c2df9a869cd10e9cd98a9390eaea2426e94

Reply:
I have implemented it in another way (f96aa3e9484e38feffc4d061d63e56415677f09c). This way it uses an API call again (JS fetch API), but works for now!

Reply:
In your first try you added this to the template:
```js
biigle.$declare('geo.overlayUrlTemplate', '{!! url('api/v1/volumes/:id/geo-overlays/geotiff/url') !!}');
```
but it should be this:
```js
biigle.$declare('geo.overlayUrlTemplate', '{!! Storage::disk(config('geo.tiles.overlay_storage_disk'))->url(':id/:id_tiles/') !!}');
```
No need to make an API call for this, fetch or otherwise. You should remove the whole API endpoint and API controller method too.

Reply:
I tried to do it like that, but I forgot to put the outer quotes, received an error, and ditched the idea. The outer quotes are necessary, since it also renders to an url, is that right?

Reply:
The outer quotes are necessary because this must be valid JavaScript and the URL should be a string.